pytorch/inference/docker/1.13/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,56 @@
+{
+  "torch": [
+    {
+      "//Reason_for_Allow": "The binaries are patched for this CVE, but the system checks version and not code.",
+      "description": " In PyTorch before trunk/89695, torch.jit.annotations.parse_type_line can cause arbitrary code execution because eval is used unsafely.", 
+      "vulnerability_id": "CVE-2022-45907", 
+      "name": "CVE-2022-45907", 
+      "package_name": "torch", 
+      "package_details": {
+        "file_path": "opt/conda/lib/python3.9/site-packages/torch-1.13.0+cpu.dist-info/METADATA", 
+        "name": "torch", 
+        "package_manager": "PYTHONPKG", "version": "1.13.0+cpu", "release": null
+      }, 
+      "remediation": {
+        "recommendation": {"text": "None Provided"}
+      }, 
+        "cvss_v3_score": 9.8, 
+        "cvss_v30_score": 0.0, 
+        "cvss_v31_score": 9.8, 
+        "cvss_v2_score": 0.0, 
+        "cvss_v3_severity": "CRITICAL", 
+        "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-45907.html", 
+        "source": "UBUNTU_CVE", 
+        "severity": "MEDIUM", 
+        "status": "ACTIVE", 
+        "title": "CVE-2022-45907 - torch"
+    }, 
+    {
+      "//Reason_for_Allow": "The binaries are patched for this CVE, but the system checks version and not code.",
```

Comment:
Same as above, can we rename this to `reason_to_ignore`

Reply:
updated